### PR TITLE
Leading section

### PR DESCRIPTION
**Pull request by Ma9da**
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -56,6 +56,116 @@ <h3><a href="#">physionic</a></h3>
...
+                    the two major realms of Classical physics: Newtonian mechanics
+                </p>
+                </p>
+            </div>
+            <div class="leading__cards">
+                <!-- card start  -->
+                <div class="leading__cards-card">
+                    <div class="leading__cards-card-header">
+                        <div class="card-icon">
+                            <svg class="person-icon" viewBox="0 0 25 20" xmlns="http://www.w3.org/2000/svg">
```
Comment (Safei-Ashraf) [inline index.html]: use img instead of svg for readbility

File: index.html [hunk 2]
```diff
@@ -56,6 +56,116 @@ <h3><a href="#">physionic</a></h3>
         </div>
     </header>
     <!-- ========== End hero ========== -->
+    <!-- ========== Start leading ========== -->
+    <div class="leading">
```
Comment (Safei-Ashraf) [inline index.html]: why are we using extra div here?

File: index.html [hunk 3]
```diff
@@ -56,6 +56,116 @@ <h3><a href="#">physionic</a></h3>
...
+        <div class="flex-container container">
```
Comment (Safei-Ashraf) [inline index.html]: make the container into <section element

File: index.html [hunk 4]
```diff
@@ -56,6 +56,116 @@ <h3><a href="#">physionic</a></h3>
...
+            <div class="section__header">
```
Comment (Safei-Ashraf) [inline index.html]: use header elem

File: index.html [hunk 5]
```diff
@@ -56,6 +56,116 @@ <h3><a href="#">physionic</a></h3>
...
+                            <a href="#">Learn More</a>
+                        </ul>
+                    </div>
+                </div>
+                <!-- card end  -->
+                <!-- card start  -->
+                <div class="leading__cards-card">
+                    <div class="leading__cards-card-header">
+                        <div class="card-icon">
+                            <svg width="25" height="20" viewBox="0 0 25 20" fill="none"
```
Comment (Safei-Ashraf) [inline index.html]: img instead of svg

File: index.html [hunk 6]
```diff
@@ -56,6 +56,116 @@ <h3><a href="#">physionic</a></h3>
...
+                                xmlns="http://www.w3.org/2000/svg">
```
Comment (Safei-Ashraf) [inline index.html]: same

**Review by Safei-Ashraf — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Ma9da) [reply index.html]: this is the div holding the whole section if the section has a bg color or need some padding

Comment (Safei-Ashraf) [reply index.html]: you can add the padding or margins on the section it self, instead of using 2 layers of un-needed divs

**Pull request merged**